_includes/footer.html

```diff
@@ -1,15 +1,18 @@
-<footer>
+<footer role="contentinfo">
+  <!-- Back to top navigation -->
+  <a class="back-to-top" href="#content">Back to top</a>
   <div class="grid-container">
     <div class="grid-x grid-padding-x">
       <div class="large-12 small-12 cell">
-        <p>©️ <a href="https://projectsbyif.com">Projects By IF</a> Ltd. 2018</p>
-        <p>Licenced under <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International</a>.</p>
-        <nav>
+        <!-- Changed copyright symbol for VO compatibility -->
+        <p>&copy <a href="https://projectsbyif.com">Projects By IF</a> Ltd. 2018</p>
+        <p>Licenced under <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International.</a></p>
+        <!-- Deleted navigation in footer as it's just external links, not site navigation. -->
+        <!-- (VO assigning it semantic meaning equivalent to pattern category menu or header navigation) -->
```

Comment:
This is really helpful for everyone else, thanks of explaining here why you made the change.